### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by kordjamshidi**
It would be good to merge this now such that the null pointer to some links is not there any more. There is no new information still but just added the files that we had pointer to them.

### CODE REVIEW

File: saul-core/doc/SAULLANGUAGE.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+## Classifiers
```
Comment (danyaljj) [inline saul-core/doc/SAULLANGUAGE.md]: This seems to be copied from `CONCEPTUALSTRUCTURE.md`. Did you mean to delete it from that file as well?

Comment (kordjamshidi) [inline saul-core/doc/SAULLANGUAGE.md]: Actually, as I said no information added, but I think we need to keep those on the main `CONCEPTUALSTRUCTURE.md` and the new files have those information again and a extension with more details and more examples later. Is that ok?

Comment (danyaljj) [inline saul-core/doc/SAULLANGUAGE.md]: I see ... the distinction was not clear to me. Good by me.

**Comment by danyaljj** [comment]
Looking good to me. Feel free to merge.

**Comment by kordjamshidi** [comment]
BTW, what did we mean to have in SaulLibrary documentation file?

**Comment by danyaljj** [comment]
I think you created based on [LBJava's structure](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/doc/LBJLIBRARY.md), but I don't think we need it. In lbjava it contains the main classes and libraries, but that's what Javadoc/Scaladoc are designed for.

**Comment by kordjamshidi** [comment]
Yes, right. Should I remove that file then?

**Comment by danyaljj** [comment]
Sounds ok to me to remove.

**Pull request merged**